github_api_helper.js

```diff
@@ -46,13 +46,21 @@ function getPRFiles (context, number) {
 /**
  * Get file contents as raw data
  * @param {import('probot').Context} context Probot context
+ *  @param {any[]} pullRequests object which contains information about the pull request
  * @param {string} path file path relative to repository root
  * @param {string} ref sha of file revision
  * @returns {Promise<any>} GitHub response
  * See https://developer.github.com/v3/repos/contents/#get-contents
  */
-function getRawFileContents (context, path, ref) {
-  const { owner, repo } = context.repo()
+function getRawFileContents (context, pullRequest, path, ref) {
```

Comment:
Rather than passing in the full PR object, add an optional head parameter so that the calling code can use the default behavior or opt-in to select a different repository than the one that sent the action.

```suggestion
function getRawFileContents (context, path, ref, head) {
```

Reply:
Agree.